assignments/objects.js

```diff
@@ -19,27 +21,89 @@ const example = {
 
 // Write your intern objects here:
 
+const example1 = {
+  "id": 1,
+  "name": "Mitzi",
+  "email": "mmelloy@psu.edu",
+  "gender": "F"
+}
+
```

Comment:
you could have also done it this way
let mitzi = {
  "id": 1,
  "name": "Mitzi",
  "email": "mmelloy0@psu.edu",
  "gender": "F"
}